src/vm-repair/azext_vm_repair/repair_utils.py

```diff
@@ -462,10 +462,13 @@ def _unlock_mount_linux_encrypted_disk(repair_vm_name, repair_group_name):
     return _invoke_run_command(LINUX_RUN_SCRIPT_NAME, repair_vm_name, repair_group_name, True)
 
 
-def _unlock_mount_windows_encrypted_disk(repair_vm_name, repair_group_name):
+def _unlock_mount_windows_encrypted_disk(repair_vm_name, repair_group_name, encrypted_vm_recovery_password):
     # Unlocks the disk using the phasephrase and mounts it on the repair VM.
     WINDOWS_RUN_SCRIPT_NAME = 'win-mount-encrypted-disk.ps1'
-    return _invoke_run_command(WINDOWS_RUN_SCRIPT_NAME, repair_vm_name, repair_group_name, False)
+    BITLOCKER_RECOVERY_PARAMS = None
+    if encrypted_vm_recovery_password is None:
```

Comment:
perhaps "if encrypted_vm_recovery_password is **not** None:"

Reply:
Fixed in latest.